deeplator/translator.py

```diff
@@ -3,7 +3,8 @@
 from .jsonrpc import JSONRPCBuilder
 
 POST_URL = "https://www2.deepl.com/jsonrpc"
-VALID_LANGS = ["EN", "DE", "FR", "ES", "IT", "NL", "PL"]
+SOURCE_LANGS = ["AUTO", "EN", "DE", "FR", "ES", "IT", "NL", "PL"]
```

Comment:
I'd like the language codes to be index independent for better readability. I think for now it's sufficient to define a `AUTO_LANG = "AUTO"` and set `SOURCE_LANGS = TARGET_LANGS + [AUTO_LANG]`.